core/rpc/src/rpc_impl/transfer.rs

```diff
@@ -104,6 +160,56 @@ where
     }
 
     pub(crate) fn inner_create_wallet(
+        &self,
+        address: String,
+        udt_info: Vec<WalletInfo>,
+        fee_rate: u64,
+    ) -> Result<TransactionCompletionResponse> {
+        let mut estimate_fee = START_ESTIMATE_FEE;
+        loop {
+            match self.inner_create_wallet_with_fixed_fee(
+                address.clone(),
+                udt_info.clone(),
+                estimate_fee,
+            ) {
+                Ok(TransactionCompletionResponse {
+                    tx_view,
+                    sigs_entry,
+                }) => {
+                    let tx_size = calculate_tx_size_with_witness_placeholder(
+                        tx_view.clone(),
+                        sigs_entry.clone(),
+                    );
+                    let actual_fee = fee_rate.saturating_mul(tx_size as u64) / 1000;
+                    if estimate_fee < actual_fee {
+                        // increase estimate fee by 1 CKB
+                        estimate_fee += BYTE_SHANNONS;
+                        continue;
+                    } else {
+                        let change_address = parse_address(&address).unwrap();
+                        match self.update_tx_view_change_cell(
+                            tx_view,
+                            change_address,
+                            estimate_fee,
+                            actual_fee,
+                        ) {
+                            Ok(tx_view) => {
+                                let adjust_response =
+                                    TransactionCompletionResponse::new(tx_view, sigs_entry);
+                                return Ok(adjust_response);
+                            }
+                            Err(_e) => {
```

Comment:
Why continue here?

Reply:
The only reason `update_tx_view_change_cell` returns an error is when the change cell is not found, then it would increse the estimate_fee and try again. The odds for not founding a change cell is almost negligible, and it would stop when estimate_fee is too high finally.

Reply:
After future investigation, it's a bug.  fixed.